opimodel/fonts.py

```diff
@@ -28,9 +32,23 @@ def __eq__(self, other):
                self.pixels == other.pixels)
         return val
 
+    def __str__(self):
+        pixels_or_points = 'px' if self.pixels else 'pt'
+        format_string = 'Font name {}: {} style {} size {}{}'
+        return format_string.format(self.name, self.fontface, self.style,
+                                    self.size, pixels_or_points)
+
+
+def parse_css_font_file(filename):
+    """
+    Parse the provided font.def file, create Font objects for each
+    defined font and attach them to the namespace of this module wth
+    names converted into appropriate constants by the
+    utils.mangle_name() function.
```

Comment:
Same as for the colours parse comments: no need to mention `utils.mangle_name` and docs need to say when this should be called.